src/graphql/node/status.rs

```diff
@@ -9,12 +9,12 @@ use roxy::ResourceUsage;
 
 use super::{
     super::{BoxedAgentManager, Role, RoleGuard},
-    ModuleName, NodeStatus, NodeStatusQuery, NodeStatusTotalCount,
+    matches_manager_hostname, AgentStatus, NodeStatus, NodeStatusQuery, NodeStatusTotalCount,
 };
 
 #[Object]
 impl NodeStatusQuery {
-    /// A list of status of all the nodes.
+    /// A list of status of nodes.
```

Comment:
Why would you remove `all the`? Do you think the list that this method returns
might not include all the nodes?

Reply:
I removed the "all" because this endpoint returns maximum 100 nodes in a single response.

Reply:
Thank you for reminding me of that.